backend/src/services/gitBookService/PageContentManager.ts

```diff
@@ -0,0 +1,50 @@
+import { RequirementStatusEnum } from "myTypes";
+
+type Requirement = {
+    status: RequirementStatusEnum;
+    requirement: string;
+};
+
+class GitBookPageContentManager {
+
+    constructor() {
+    }
+
+    extractRequirements(pageContent): Requirement[] {
+        if (!pageContent || !pageContent.document || !pageContent.document.nodes) {
```

Comment:
you can shorten it using if (!(pageContent?.document?.nodes)) syntax

Reply:
Done